lightning/src/ln/wire.rs

```diff
@@ -464,6 +482,19 @@ impl Encode for msgs::AcceptChannelV2 {
 	const TYPE: u16 = 65;
 }
 
+impl Encode for msgs::Splice {
+	// TODO(splicing) Double check with spec; spec contains 74, which is probably wrong as it is used by tx_Abort; CLN uses 75
+	const TYPE: u16 = 75;
```

Comment:
Seems fine for now with CLN using it. I think t-bast's comment [here](https://github.com/lightning/bolts/pull/863/files#r1323163643) is somewhat valid around leaving space for extending interactive txs and rather starting at 80 but let's just keep it like this for now.